tests/helper_test.php

```diff
@@ -41,6 +41,13 @@ public function setUp(): void {
         $this->generator = $this->getDataGenerator()->get_plugin_generator('mod_hvp');
     }
 
+    /**
+     * Tear down after every test.
+     */
+    protected function tearDown(): void {
+        $this->generator = null;
```

Comment:
could it better just to use `unset($this->generator);` instead of explicitly setting it to null?

Reply:
Hi @jwalits ,

Thank you for the suggestion.
Updated the changes.